doc/ru/Privacy.md

```diff
@@ -0,0 +1,28 @@
+# Политика конфиденциальности
+
+Следующая информация относится к RHVoice, синтезатору речи, разработанному
+Ольгой Яковлевой. Его исходный код опубликован по адресу
+<https://github.com/RHVoice/RHVoice>.
+
+RHVoice может использовать сторонние сервисы, которые собирают личную
+информацию. Они будут перечислены в этом документе с необходимыми пояснениями.
+Если вы используете соответствующую функциональность RHVoice, вы соглашаетесь с
+условиями и положениями, изложенными в политике конфиденциальности этих
+сервисов.
+
+Сам RHVoice не собирает никакой личной информации.
+
+После того, как RHVoice преобразует фрагмент текста в речь, он забывает об этом
+тексте: никакая часть текста или сгенерированная речь не сохраняется RHVoice.
+
+Синтез речи выполняется локально, на устройстве пользователя, без передачи
+какой-либо части текста или сгенерированной речи на удаленный сервер.
+
+Приложение RHVoice для Android использует
+[Linode Object Storage](https://www.linode.com/products/object-storage/),
+сторонний хостинг-сервис для размещения файлов языковых и голосовых данных.
+Когда вы решите установить голос, и RHVoice будет загружать необходимые файлы
+из Object Storage, Linode может собирать ваши метаданные, такие как ваш

```

Comment:
May make sense to mention that anonimizer software may be helpful.

Reply:
What? Please create your PR for changes doc/en/privacy.md.